corner_rows: 5
corner_cols: 8
cornersize_rows: 7.23
cornersize_cols: 7.23
fold_path: "/home/yang/Dataset/image_fold3"
line_image_path: "/home/yang/Dataset/image_fold3/light_fold"
intrinsic_file: "../config/intrinsic.yaml"
camera_model: 1
